data/strings/strings.txt

```diff
@@ -20554,6 +21022,7 @@
     ja = 検索を使うか、地図上をタップしてルートの出発点を追加する
     ko = 검색을 사용하거나 지도를 탭하여 경로 시작점을 추가합니다
     lt = Naudokite paiešką arba bakstelėkite žemėlapyje, kad pridėtumėte maršruto pradžios tašką
+    lv = Lietojiet meklēšanu vai uzsitiet uz kartes, lai pievienotu sākuma punktu
```

Comment:
"uzsitiet" -> "pieskarieties"

Reply:
And then also "uz kartes" -> "kartei".

Reply:
Thanks —
"Lietojiet meklēšanu vai pieskarieties kartei, lai pievienotu sākuma punktu"